fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/AbstractMaterializedViewAggregateRule.java

```diff
@@ -284,6 +285,26 @@ protected Plan rewriteQueryByView(MatchMode matchMode,
         }
         // add project to guarantee group by column ref is slot reference,
         // this is necessary because physical createHash will need slotReference later
+        for (Expression expression : queryChildrenGroupBySet) {
```

Comment:
this can add if condition as following for performance
        if (!queryGroupByExpressions.isEmpty() && queryTopPlanGroupBySet.isEmpty()) {
           do your check logic
        }

Reply:
有没有可能queryTopPlanGroupBySet不为空，但是也没有包含全部的groupby字段 这种情况？
比如 `select sum(o_totalprice), a from orders group by o_orderdate, a, b`
这种情况的话`if (!queryGroupByExpressions.isEmpty() && queryTopPlanGroupBySet.isEmpty())`是不是筛查不出来

Reply:
you are right, this check condition should be  `if top group by doest equals bottom group by, we should add check`